examples/disruption_crons/testing.yaml

```diff
@@ -0,0 +1,21 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: DisruptionCron
+metadata:
+  name: network-drop-147
+  namespace: chaos-demo
+spec:
+  schedule: "*/15 * * * *"
+  targetResource:
+    kind: deployment
+    name: chaos-microservices-frontend
```

Comment:
I think we only want to commit yamls that work with our example demo?

Reply:
now how did that get in there >:(